build/docker/template_hazelcast.xml

```diff
@@ -23,18 +18,16 @@
         </outbound-ports>
         <join>
             <multicast enabled="false"/>
-            <aws enabled="false"/>
+            <aws enabled="true">
+                <region>eu-central-1</region>
```

Comment:
Ok, we need to inject this one too. I'll write a ticket.

Reply:
Actually, do you even have to specify it?
First of all, the default is probably to use the region the Fargate task is running on.
And secondly, in the future, the cluster will span multiple regions, so you cannot only join the ones in your region.

Reply:
consider it removed